src/sdk/pynni/nni/trial.py

```diff
@@ -50,11 +50,15 @@ def get_next_parameter():
         return None
     return _params['parameters']
 
-def get_current_parameter(tag):
+def get_current_parameter(tag=None):
     global _params
     if _params is None:
         return None
-    return _params['parameters'][tag]
+    if tag is None:
+        # return all parameters when tag is None
+        return _params["parameters"]
```

Comment:
not necessary

Reply:
This is because I have to retrieve all the parameters to do the parsing. `trials._params` seems not a good practice since variables starting with underline is by convention seen as private to module.

Reply:
let's discuss in the meeting